pkg/api/v1alpha1/tinkerbelldatacenterconfig_types.go

```diff
@@ -18,7 +18,7 @@ type TinkerbellDatacenterConfigSpec struct {
 	// It must include the Kubernetes version(s). For example, a URL used for Kubernetes 1.27 could
 	// be http://localhost:8080/ubuntu-2204-1.27.tgz
 	//+optional
-	OSImageURL string `json:"osImageURL,omitempty"`
+	OSImageURL string `json:"osImageURL"`
```

Comment:
we shouldn't change this since we are still supporting br where its okay for this to be empty right?